lib/diff.mli

```diff
@@ -7,6 +7,13 @@ type value = {
     (Types.value_description, Types.value_description atomic_modification) t;
 }
 
+type class_ = {
+  cname : string;
+  cdiff : (Types.class_declaration, class_modification) t;
+}
+
+and class_modification = Added | Removed | Modified
```

Comment:
Type `t` already handles addition and deletion, so there is not point in having them as cases in `class_modification`.

This PR does not deal with the modification case yet (which is on purpose!), so this type is not used. To make this explicit, I suggest to define it as a single constructor: `Unsupported`.

The `class_modification` type will, when we